.claude/skills/ticketing/SKILL.md

```
@@ -54,27 +54,79 @@ wb_document_set   → { workspaceId: "default", canvasId, markdown: "<updated OK
 wb_facet_set → { workspaceId: "default", canvasId, facets: { "<namespace>.<name>/v0": { … } }  # key grammar per ADR-0013 }
 ```
 
-### Resolving / deleting
+### Resolving
+
+**Resolution is `type: issue` -> `type: note`, not deletion.** Re-import the
+document with the type changed, the name prefixed `RESOLVED — `, and a line
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)